examples/src/main/python/ml/simple_params_example.py

```diff
@@ -36,18 +35,20 @@
     if len(sys.argv) > 1:
```

Comment:
I see.. Hm.. but I don't think this is meaningless but means explicitly not taking arguments. 

Actually, I think all the examples (not taking arguments) should check this for consistency because some of example scripts (taking arguments) are already checking this.

Strictly, running this example with arguments might not be a proper way to run this example.